app/javascript/packs/application.js

```diff
@@ -1,13 +1,16 @@
 import Vue from 'vue/dist/vue.esm'
 import system from 'lux-design-system'
-import 'lux-design-system/lib/system/system.css'
+import 'lux-design-system/dist/system/system.css'
 import store from '../store'
 import DocumentAdder from '../components/document_adder'
 import PlaylistMembers from '../components/playlist_members'
 import axios from 'axios'
+import OrderManager from '../components/OrderManager.vue'
 
 Vue.use(system)
 
+Vue.component('order-manager', OrderManager)
```

Comment:
How do you feel about https://frontendsociety.com/why-you-shouldnt-use-vue-component-ff019fbcac2e?

Reply:
I don't think there will be an issue with global conflicts of the OrderManager component, but there's some good points in the article and certainly no reason _not_ to make it local. 👍